modules/src/applications/ics20_fungible_token_transfer/events.rs

```diff
@@ -116,6 +117,22 @@ impl From<DenomTraceEvent> for ModuleEvent {
     }
 }
 
+pub struct TransferEvent {
```

Comment:
Maybe we need add transfer denom and amount.

Reply:
I instead chose to log this to stay consistent with ibc-go -> https://github.com/informalsystems/ibc-rs/pull/1989/files#diff-3dee6bfbafa486af143dfe453c7cc29776865fc1da85b431cbf5fb57eaa637b9R114-R117